packages/proto-loader/src/index.ts

```diff
@@ -395,6 +395,23 @@ export function loadSync(
   return createPackageDefinition(root, options!);
 }
 
+export function fromJSON(
+  json: Protobuf.INamespace,
+  root?: Protobuf.Root
```

Comment:
Protobuf.js objects do not belong in this API. The `root` argument should be removed.